packages/sui-react-web-vitals/src/index.js

```diff
@@ -57,6 +57,12 @@ export default function WebVitalsReporter({
   }, [onReport])
 
   useMount(() => {
+    const {
+      deviceMemory,
+      connection: {effectiveType},
```

Comment:
This will break on Safari and Firefox.
navigator.connection it's not implemented yet on these browsers.
![imatge](https://github.com/SUI-Components/sui/assets/17271365/1776dc23-409f-420d-8a76-34f1741abc1e)

Reply:
Updated